### PR TITLE
feat: adding buildargs support

### PR DESCRIPTION
**Pull request by geirearnesen**
Adding support for passing buildargs to the Docker build process.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,13 @@ The Deploy action performs the functions of combining the [Build](https://github
 **Optional** Whether to utilise the docker cache during the build. Defaults to true.
 ### `manifest`
 **Optional** Use a custom path for your convox.yml
+### `buildargs`
+**Optional** Pass docker build arguments to the build process. Build args are key-value pairs passed as a list like:
+```
+  build-args:
```
Comment (numcys) [inline README.md]: The input key is 'buildargs' in the action.yml file.

File: README.md [hunk 2]
```diff
@@ -15,6 +15,13 @@ The Deploy action performs the functions of combining the [Build](https://github
...
+     - BASEIMAGE=myimage
```
Comment (numcys) [inline README.md]: Github action does not support list input. You can either make it a string, where build arguments are seperated by a newline('\n') character.
ex:
buildargs: "key1=value1 \n key2=value2"

File: entrypoint.sh [hunk 3]
```diff
@@ -22,4 +22,18 @@ if [ "$INPUT_MANIFEST" != "" ]; then
     MANIFEST_COMMAND="-m $INPUT_MANIFEST"
 fi
 
-convox deploy --app $INPUT_APP --description "$INPUT_DESCRIPTION" $CACHED_COMMAND $MANIFEST_COMMAND
+# Split the INPUT_BUILDARGS by newline into an array
+if [ "$INPUT_BUILDARGS" != "" ]; then
+    IFS=$'\n' read -d '' -r -a ADDR <<< "$INPUT_BUILDARGS"  # Split build arguments by newline
```
Comment (numcys) [inline entrypoint.sh:27]: The '<<<' redirection is not supported in /bin/sh shell. You can change the shell to /bin/bash at the top of the file.

File: README.md (hunk 1 above)
Comment (geirearnesen) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (geirearnesen) [reply README.md]: Fixed

**Review by numcys — APPROVED** [review]

**Review by nightfury1204 — APPROVED** [review]
@geirarnesen Thanks for creating the PR.
lgtm

**Comment by nightfury1204** [comment]
@geirarnesen released it: https://github.com/convox/action-deploy/releases/tag/2.0.1

**Pull request merged**